src/components/CurrencyInput.tsx

```diff
@@ -89,7 +89,7 @@ export const CurrencyInput: FC<CurrencyInputProps> = forwardRef<
       groupSeparator,
       disableGroupSeparators,
       intlConfig,
-      prefix: prefix || localeConfig.prefix,
+      prefix: prefix ?? localeConfig.prefix,
       suffix: suffix,
```

Comment:
Can we also add support for `suffix` too? @cchanxzy do you think we could merge this PR? It would be useful to have this option